tools/transformer/src/transformations.js

```diff
@@ -133,13 +129,18 @@ function transformAllOf(objectName, parentObject, api) {
       if (!child.allOf) {
         child.allOf = {};
       }
-      
       // Expand parent in child allOf
       child.allOf.push(expandedParent);
       // Flatten child allOf
-      flattenAllOfObject(child, api);
     }
   }
+
+  // Remove invalid fields
+  parentObject.properties = null;
+  delete parentObject.properties;
+
+  parentObject.required = null;
```

Comment:
that would be affecting client-side validation. Is that desired?

Reply:
I see that it was moved, just wondering

Reply:
I was trying to figure out a bug, and I thought that I was deleting by mistake some properties and those would be garbage collected. I will revert those changes

Reply:
Actually you are right. If properties are required but we remove them openapi schema will no longer be valid.